spec/message.abnf

```diff
@@ -94,6 +73,19 @@ name-start = ALPHA / "_"
 name-char  = name-start / DIGIT / "-" / "."
            / %xB7 / %x300-36F / %x203F-2040
 
+content-char = %x00-08        ; omit HTAB (%x09) and LF (%x0A)
+             / %x0B-0C        ; omit CR (%x0D)
+             / %x0E-19        ; omit SP (%x20)
+             / %x21-2D        ; omit . (%x2E)
+             / %x2F-5B        ; omit \ (%x5C)
+             / %x5D-7A        ; omit { | } (%x7B-7D)
+             / %x7E-D7FF      ; omit surrogates
+             / %xE000-10FFFF
```

Comment:
As above, move the `=` and `/` out to match the *-char productions under this.

I think maybe a comment is well advised here, along the lines of:

```abnf
; base repertoire of characters for various text productions
```

Side note: we haven't done a great job of this, but a common practice in ABNF grammars is to put constituent parts _under_ where they are used (that is, forward definitions are the norm).

In this case, that suggests putting the *-char definitions first and then this item that they all consume.

Reply:
Moved the `content-char` definition below its users.